pydantic_ai_slim/pydantic_ai/models/bedrock.py

```diff
@@ -420,13 +436,7 @@ async def _map_messages(
                                 }
                             )
             elif isinstance(message, ModelResponse):
-                content: list[ContentBlockOutputTypeDef] = []
-                for item in message.parts:
-                    if isinstance(item, TextPart):
-                        content.append({'text': item.content})
-                    else:
-                        assert isinstance(item, ToolCallPart)
-                        content.append(self._map_tool_call(item))
+                content: list[ContentBlockOutputTypeDef] = self._map_model_response(message)
                 bedrock_messages.append({'role': 'assistant', 'content': content})
```

Comment:
It's now possible for `content` to be an empty list, in which case we should not append message at all, right?